.swiftlint.yml

```diff
@@ -0,0 +1,6 @@
+opt_in_rules:
+    - force_unwrapping
```

Comment:
Please disable the `trailing_whitespace` rule, unless there’s a way for it to ignore blank lines that are indented, such as:

```swift
    let foo: String?
    
    let bar: String?
```

We should use the linter for legitimate issues that affect product behavior or maintainability. I can’t think of a single time we’ve ever nitpicked about trailing whitespace on a PR in this repository. Meanwhile, indenting blank lines is the Xcode default.

If we err on the side of passing, we can keep the barrier to entry low for new contributors while making it more likely that they’ll get a 👍 on their first try.